modules/identity-provider-configuring-apache-request-header.adoc

```diff
@@ -1,100 +1,82 @@
-2. Enforce client certificate authentication in the `RequestHeaderIdentityProvider` configuration.
-3. Require the `X-Csrf-Token` header be set for all authentication request using the challenge flow.
-4. Only the `/oauth/authorize` endpoint and its subpaths should be proxied,
+. Block the `X-Remote-User` header from client requests to prevent spoofing.
+. Enforce client certificate authentication in the `RequestHeaderIdentityProvider` configuration.
+. Require the `X-Csrf-Token` header be set for all authentication requests using the challenge flow.
+. Only the `/oauth/authorize` endpoint and its subpaths should be proxied,
 and redirects should not be rewritten to allow the backend server to send the client to the correct
 location.
-5. The URL that proxies to `\https://<master>/oauth/authorize` must end with `/authorize` (with no trailing slash). For example:
-  * `\https://proxy.example.com/login-proxy/authorize?...` -> `\https://<master>/oauth/authorize?...`
-6. Subpaths of the URL that proxies to `\https://<master>/oauth/authorize` must proxy to subpaths of `\https://<master>/oauth/authorize`. For example:
-  * `\https://proxy.example.com/login-proxy/authorize/approve?...` -> `\https://<master>/oauth/authorize/approve?...`
-
-. Obtain the `mod_auth_gssapi` module from the
-link:https://access.redhat.com/solutions/392003[Optional channel].
-Install the following packages:
+. The URL that proxies to `\https://<namespace_route>/oauth/authorize` must end with `/authorize` (with no trailing slash). For example:
 +
 ----
-# yum install -y httpd mod_ssl mod_session apr-util-openssl mod_auth_gssapi
+\https://proxy.example.com/login-proxy/authorize?...
 ----
-
-. Generate a CA for validating requests that submit the trusted header. Use this CA
-as the file name for `clientCA` in the identity provider configuration.
++
+should proxy to
 +
 ----
-# oc adm ca create-signer-cert \
-  --cert='/etc/origin/master/proxyca.crt' \
-  --key='/etc/origin/master/proxyca.key' \
-  --name='openshift-proxy-signer@1432232228' \
-  --serial='/etc/origin/master/proxyca.serial.txt'
+\https://<namespace_route>/oauth/authorize?...
 ----
 +
+. Subpaths of the URL that proxies to `\https://<namespace_route>/oauth/authorize` must proxy to subpaths of `\https://<namespace_route>/oauth/authorize`. For example:
++
+----
+\https://proxy.example.com/login-proxy/authorize/approve?...
+----
++
+should proxy to
++
+----
+\https://<namespace_route>/oauth/authorize/approve?...
+----
+
 [NOTE]
 ====
-The `oc adm ca create-signer-cert` command generates a certificate that is valid
-for five years. This can be altered with the `--expire-days` option, but for
-security reasons, it is recommended to not make it greater than this
-value.
-
-// Run `oc adm` commands only from the first master listed in the Ansible host inventory file,
-// by default *_/etc/ansible/hosts_*.
+The `https://<namespace_route>/` address is the location where `.well-known`
```

Comment:
This actually generates a clickable link :frowning_face:  Is there something that we can do to prevent this which would not add an extra character on the resulting page?

Reply:
`https://<namespace_route>/` is the route to the oauth server and can be obtained by `oc get route -n openshift-authentication`, since we're talking about the integrated oauth server